chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/results/CertificationIdResults.java

```diff
@@ -50,6 +54,8 @@ public void setProductId(final Long id) {
     private String ehrCertificationId;
     private Map<String, Integer> metCounts;
     private Map<String, Integer> metPercentages;
+    private SortedSet<String> missingAnd;
```

Comment:
Any particular reason to use SortedSet or TreeSet? Is anything actually being sorted here?